src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -1463,7 +1463,7 @@ public MessageSticker createSticker(DataObject content)
     {
         final long id = content.getLong("id");
         final String name = content.getString("name");
-        final String description = content.getString("description");
+        final String description = content.getString("description", null);
```

Comment:
Could we make this backwards compatible for now by defaulting to an empty string?

Reply:
yes

![](https://cdn.discordapp.com/emojis/741207256913149972.png)